tests/Unit.Tests.Target.Net5/Log4NetLoggerTests.cs

```diff
@@ -180,6 +181,66 @@ public void Log_Should_IgnoreMessage_With_LevelNone()
                         .BeEmpty();
         }
 
+        [Fact]
+        public void Log_Should_Emit_LoggingEvents_Created_By_Custom_LoggingEventFactory()
+        {
+            var options = ConfigureOptions(Log4NetFileOption.TestAppenderTrace);
+
+            var callStackBoundary = typeof(Log4NetLoggerTests);
+            var exspectedRepository = log4net.LogManager.GetRepository(options.LoggerRepository);
```

Comment:
Minor misspelling. `expectedRepository`. And so on...

Sure that you can found more on existing code, but... you know. :)

Reply:
fixed